src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlGuidTranslator.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Linq.Expressions;
+using System.Reflection;
+using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore.Query.Expressions;
+using Microsoft.EntityFrameworkCore.Query.ExpressionTranslators;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query.ExpressionTranslators.Internal
+{
+    /// <summary>
+    /// Provides translation services for PostgreSQL UUID functions.
+    /// </summary>
+    /// <remarks>
+    /// See: https://www.postgresql.org/docs/current/datatype-uuid.html
+    /// </remarks>
+    public class NpgsqlGuidTranslator : IMethodCallTranslator
```

Comment:
I think it may be good to copy-paste [the SqlServer implementation](https://github.com/aspnet/EntityFrameworkCore/blob/10a8ddf1d7ea5aecbd69336c4e57122cb5a68bc0/src/EFCore.SqlServer/Query/ExpressionTranslators/Internal/SqlServerNewGuidTranslator.cs#L19) - extending `SingleOverloadStaticMethodCallTranslator` makes the class slightly simpler etc.